index.js

```diff
@@ -1,4 +1,14 @@
 'use strict';
 
-console.log('Hello, world!');
+const express = require('express');
+const app = express();
 
+// Define routes
+app.get('/health', (req, res) => {
+  res.json({ message: 'OK' });
+});
+
+// Start the server
+app.listen(3000, () => {
```

Comment:
We should change the port to something other than 3000, since we use this port for the app as well. maybe 3010?

Reply:
recent commit should fix this